code/datums/character_preview.dm

```diff
@@ -106,6 +106,9 @@ datum/character_preview
 		src.preview_mob.real_name = "clone of " + name
 		src.preview_mob.name = "clone of " + name
 
+	proc/mirror_appearance(mob/living/carbon/human/to_copy) // Be careful with runtimes when using this proc.
```

Comment:
This comment is somewhat concerning. Could you expand on it (i.e. the "why")?

Reply:
I tried to explain why it's not safe to use this proc. Tell me if I should change the warning.